ios/bottom-tabs/host/RNSBottomTabsHostComponentView.mm

```diff
@@ -357,7 +378,7 @@ - (void)mountingTransactionWillMount:(const facebook::react::MountingTransaction
 - (void)mountingTransactionDidMount:(const facebook::react::MountingTransaction &)transaction
                withSurfaceTelemetry:(const facebook::react::SurfaceTelemetry &)surfaceTelemetry
 {
-  if (_hasModifiedReactSubviewsInCurrentTransaction) {
+  if (_hasModifiedTabsScreensInCurrentTransaction || _hasModifiedBottomAccessoryInCurrentTransation) {
```

Comment:
Maybe worth extracting this to separate property? You could declare an extension on the `RNSBottomTabsHostComponentView` *in this source* file & add a readonly computed property.